odxtools/fwsignature.py

```diff
@@ -0,0 +1,30 @@
+# SPDX-License-Identifier: MIT
+from dataclasses import dataclass
+from typing import cast
+from xml.etree import ElementTree
+
+from .exceptions import odxraise, odxrequire
+from .odxdoccontext import OdxDocContext
+from .sessionsubelemtype import SessionSubElemType
+
+
+@dataclass(kw_only=True)
+class FwSignature:
+    value: str
+    value_type: SessionSubElemType
```

Comment:
Duplicate with FwChecksum

Reply:
yep. the spec does it like this though. maybe this should be made a type alias...

Reply:
Other than this, LGTM

Reply:
@andlaus this topic is still open, any plans to fix it?

Reply:
for the same reason as for the `StrEnum` change I intended to do this in a separate PR. Anyway, I added the patch for this: https://github.com/mercedes-benz/odxtools/pull/422/commits/31a8abba2d6ea4e926269a93f2dfad3027bedb04

Reply:
Unfortunately I had to revert this change: `StrEnum` was only introduced in python 3.11...